apiclient/census.go

```diff
@@ -17,6 +24,20 @@ type CensusProof struct {
 	KeyType models.ProofArbo_KeyType
 }
 
+// CensusProofZk represents the proof for a voter in a zk census. It contains
+// the encoded proof and the circuit public tokens to verify it. It also keeps
+// the weight of the vote and the key type of the merkle tree, and also includes
+// the voter's nullifier. This last parameter is necessary because now, to
+// compute the nullifier, the private key of the ZkAddress is required, so it
+// cannot be computed by anyone but the voter.
+type CensusProofZk struct {
+	Proof      types.HexBytes
+	PubSignals types.HexBytes
+	Weight     uint64
```

Comment:
I think weight should be a `types.BigInt`

Reply:
The old proof parameter was uint64, should I change both types?